apps/prairielearn/elements/pl-number-input/pl-number-input.py

```diff
@@ -284,22 +297,19 @@ def render(element_html, data):
 
         partial_score = data["partial_scores"].get(name, {"score": None})
         score = partial_score.get("score", None)
+
         if score is not None:
-            try:
-                score = float(score)
-                if score >= 1:
-                    html_params["correct"] = True
-                elif score > 0:
-                    html_params["partial"] = math.floor(score * 100)
-                else:
-                    html_params["incorrect"] = True
-            except Exception:
-                raise ValueError("invalid score" + score)
+            score_type, score_value = pl.determine_score_params(score)
+            html_params[score_type] = score_value
 
         html_params["error"] = html_params["parse_error"] or html_params.get(
             "missing_input", False
         )
 
+        feedback = partial_score.get("feedback", None)
```

Comment:
```suggestion
        html_params["feedback"] = partial_score.get("feedback", None)
```
Nit: `None` is "falsy" with respect to chevron rendering, so there's no need for the if guard below

Reply:
Changed as suggested in [175f82c](https://github.com/PrairieLearn/PrairieLearn/pull/7934/commits/175f82c20eab5ac9efc8e19696f5669740fc160c).